using-directus-as-a-baby-health-tracker/index.md

```diff
@@ -0,0 +1,403 @@
+
+First, let's point out which Collections we need to save the data later on. To make it simple, we start with only two Collections. From a data-structure point of view, this might not be the best solution, but as Directus is very flexible, we can optimize the structure later on if needed.
+#### The data storage collection
+As we want to track sensor data over time, we will frequently pull the data with some details from the external API. To store this data we are using a Collection called "sensor_data".
+For the ID we use an auto-generated UUID that can be used as an index and to refer to any item in our collection later on. Using a regular auto-increment would work of cause as well.
+There are a couple of fields provided by Directus, during the creation of the new Collection we've selected the following ones:
+* sort
+* status
+* date_created
+For each element of the data provided by the sensor, we want to save these values. Depending on the used sensor, you may come up with a different set of fields. In our case we've created the following important additional fields:
+
+| Field name | Interface | Comment |
+| ---- | ---- | ---- |
+| raw_payload | Code | We use this to store the JSON payload that we gather from the external API. This is more or less just for debugging and can be skipped of cause. |
+| temperature | Input, Decimal | Just the plain value of the sensor data |
+| oxygen | Input, Decimal | Just the plain value of the sensor data |
+| heart_rate | Input, Decimal | Just the plain value of the sensor data |
+| battery_level | Input, Integer | Just the plain value of the sensor data |
+| battery_status | Dropdown | To evaluate the sensor data, a simple Dropdown field is used to display the status for each value. |
+| heart_status | Dropdown | To evaluate the sensor data, a simple Dropdown field is used to display the status for each value. |
+| oxygen_state | Dropdown | To evaluate the sensor data, a simple Dropdown field is used to display the status for each value. |
+| thermal_state | Dropdown | To evaluate the sensor data, a simple Dropdown field is used to display the status for each value. |
+#### The alerting collection (optional)
+The second Collection is only needed for the integration into OpsGenie later on, so you may skip this part if not needed in your scenario or replace/adjust it with any other structure (fields in this collection) that might become handy for your kind of reporting or integration into external tools.
+
+| Field name | Interface | Comment |
+| ---- | ---- | ---- |
+| id |  |  |
+| external_id |  |  |
+| sort |  |  |
+| user_created |  |  |
+| user_updated |  |  |
+| date_created |  |  |
+| date_updated |  |  |
+| category |  |  |
+| type |  |  |
+| severity |  |  |
+| status |  |  |
+| priority |  |  |
+| message |  |  |
+| alias |  |  |
+| details |  |  |
+| entity |  |  |
+| description |  |  |
+| neboo_data |  |  |
+| links |  |  |
```

Comment:
Obviously need to finish this off. Also confused as to whether this is a post about the Neboo or the Owlet. I understand it changed, but this whole article should focus on one.

Reply:
I've reduced the fields as not all of them are needed or mission critical. A shore "check out Opsgenie docs" should be fine I guess?